dockerfiles/init/modules/openshift/files/scripts/che-config-map

```diff
@@ -0,0 +1,34 @@
+# To put some extra CHE server configurations just add here CHE ENV vars following format:
+# KEY: "value"
+CHE_API: ${HTTP_PROTOCOL}://che-${OPENSHIFT_NAMESPACE_URL}/api
+CHE_DEBUG_SERVER: ${CHE_DEBUG_SERVER}
+CHE_HOST: che-${OPENSHIFT_NAMESPACE_URL}
+CHE_INFRASTRUCTURE_ACTIVE: openshift
+CHE_INFRA_OPENSHIFT_BOOTSTRAPPER_BINARY__URL: ${HTTP_PROTOCOL}://che-${OPENSHIFT_NAMESPACE_URL}/agent-binaries/linux_amd64/bootstrapper/bootstrapper
+CHE_INFRA_OPENSHIFT_MACHINE__START__TIMEOUT__MIN: "5"
+CHE_INFRA_OPENSHIFT_MASTER__URL: ""
+CHE_INFRA_OPENSHIFT_OAUTH__TOKEN: ${CHE_INFRA_OPENSHIFT_OAUTH__TOKEN}
+CHE_INFRA_OPENSHIFT_PASSWORD: ${CHE_INFRA_OPENSHIFT_PASSWORD}
+CHE_INFRA_OPENSHIFT_PROJECT: ${CHE_INFRA_OPENSHIFT_PROJECT}
+CHE_INFRA_OPENSHIFT_PVC_STRATEGY: ${CHE_INFRA_OPENSHIFT_PVC_STRATEGY}
+CHE_INFRA_OPENSHIFT_TLS__ENABLED: ${ENABLE_SSL}
+CHE_INFRA_OPENSHIFT_TRUST__CERTS: "false"
+CHE_INFRA_OPENSHIFT_USERNAME: ${CHE_INFRA_OPENSHIFT_USERNAME}
+CHE_KEYCLOAK_AUTH__SERVER__URL: ${CHE_KEYCLOAK_AUTH__SERVER__URL}
+CHE_KEYCLOAK_CLIENT__ID: ${CHE_KEYCLOAK_CLIENT__ID}
+CHE_KEYCLOAK_GITHUB_ENDPOINT: ${KEYCLOAK_GITHUB_ENDPOINT}
+CHE_KEYCLOAK_OSO_ENDPOINT: ${CHE_KEYCLOAK_OSO_ENDPOINT}
+CHE_KEYCLOAK_REALM: ${CHE_KEYCLOAK_REALM}
+CHE_LOCAL_CONF_DIR: /etc/conf
+CHE_LOGS_DIR: /data/logs
+CHE_LOG_LEVEL: ${CHE_LOG_LEVEL}
+CHE_MULTIUSER: ${CHE_MULTIUSER}
+CHE_OAUTH_GITHUB_CLIENTID:
+CHE_OAUTH_GITHUB_CLIENTSECRET:
+CHE_OAUTH_GITHUB_FORCEACTIVATION: "true"
+CHE_PORT: "8080"
+CHE_PREDEFINED_STACKS_RELOAD__ON__START: ${CHE_PREDEFINED_STACKS_RELOAD}
+CHE_SERVER_JAVA_OPTS: "-XX:+UseG1GC -XX:+UseStringDeduplication -XX:MinHeapFreeRatio=20 -XX:MaxHeapFreeRatio=40 -XX:MaxRAM=700m -Xms256m"
+CHE_WEBSOCKET_ENDPOINT:  ${WS_PROTOCOL}://che-${OPENSHIFT_NAMESPACE_URL}/api/websocket
```

Comment:
I'd move declaration of this env var somewhere near [CHE_API](https://github.com/eclipse/che/pull/7692/files#diff-1e554eb231a824102dbb255a1ee83d0fR3) because they are related

Reply:
can you please specify which exactly ENV you are talking about

Reply:
I'm talking about grouping of env var like it was before
```
CHE_HOST: che-${OPENSHIFT_NAMESPACE_URL}
CHE_API: ${HTTP_PROTOCOL}://che-${OPENSHIFT_NAMESPACE_URL}/api
CHE_WEBSOCKET_ENDPOINT:  ${WS_PROTOCOL}://che-${OPENSHIFT_NAMESPACE_URL}/api/websocket
```

Reply:
Done